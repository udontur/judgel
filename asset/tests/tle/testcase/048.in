1669 185
93889 6079 68123 62968 16430 17119 64514 18809 17072 34113 59628 61476 1925 43369 66679 47581 2230 46877 31060 81091 44000 1349 98342 45264 44135 11077 73254 51995 18841 10963 30574 81298 99055 99242 49263 20593 55236 94857 89173 28516 91699 70787 42752 32768 25984 36468 22829 36579 9331 70595 9161 60359 57215 6186 46358 66168 86342 22904 91124 82710 48517 51544 30196 37781 70221 10665 84228 27670 53140 36094 21562 53002 28389 71745 44011 84561 69088 94934 68114 8012 96069 34556 7282 72378 77767 34388 14514 54428 92892 48760 25447 75017 68383 1269 56614 39273 1136 98389 80679 55130 47861 18780 404 42526 31558 39181 80198 29992 85753 31536 54521 14741 74798 79554 95514 7905 90978 13418 69810 72908 65416 15962 95527 85783 81954 56041 59168 32294 45597 62352 54229 89014 43036 93509 34376 19685 96640 90112 66694 26943 13062 28175 1288 55437 61530 78242 1856 18122 496 33770 46552 29372 82560 26788 43826 15619 22103 60892 32738 59561 24777 54365 41256 84692 70561 46078 90307 26937 72797 3561 2048 53203 68218 94257 71445 13165 75544 95875 17966 2513 64766 48737 2709 13797 42714 96207 672 12936 3837 36280 77867 105 93479 62154 1331 29569 11203 62616 38225 71613 18979 78844 35725 73263 21884 49641 61701 32237 38827 85198 17331 87484 28978 20094 91947 9833 24067 78198 23679 917 6459 37130 370 16460 45878 45611 64977 88875 67828 59864 68536 87331 74736 38357 9516 87364 1270 18256 35793 54911 31069 5612 31001 21890 88153 73008 31558 12181 8472 62101 23546 65170 76125 18780 29150 80041 83193 73092 80244 47206 91701 49960 22545 97970 54035 87340 61141 15760 67670 64732 56669 81223 5740 89522 80411 99603 2432 11956 83478 79993 31455 46132 43352 18755 30996 22196 51724 99519 37213 12800 14914 88886 20785 51418 4229 29445 32281 86137 83739 89798 72720 43450 18015 65026 17667 5749 15127 24259 38089 21257 60453 78172 17055 56384 4443 10798 84223 18145 18192 11868 66112 15920 1475 78340 45330 76178 41214 22051 30395 57675 37072 40714 73612 1498 63566 20781 21012 69772 70557 92178 54425 93911 20571 16268 37262 97190 3549 82803 99727 30431 93979 12654 12457 96733 13988 79376 19301 16066 39289 5567 58828 10280 76375 51014 4802 86576 91723 42043 85659 74812 40745 37802 49652 69907 28785 72196 42419 19160 65429 66944 203 57456 95190 46029 89099 41379 54249 4999 61087 30918 72788 37249 31362 53571 91005 94168 13315 96881 58332 44816 28159 37955 34315 70497 1399 25557 69275 63893 7024 74297 94369 30956 81958 55329 32666 62945 9164 1429 79559 94310 34836 49755 342 77216 72991 56122 80614 5467 69451 17194 58268 97861 88705 9665 85286 79612 74767 77762 59360 91367 85039 70084 14265 89607 66283 36716 24628 93929 41531 32349 46415 43692 53456 76153 11638 67629 31541 71372 18863 71521 58360 32519 82681 24420 40734 87243 99169 25634 83469 27591 63325 3939 6656 53250 36706 28270 26707 13779 27111 83041 38999 28498 86112 61403 93575 44420 8777 1925 4214 46167 15233 99595 95563 94192 31922 94617 11119 23613 90185 66991 93062 636 18607 99590 30252 97046 26843 83189 2699 10251 62129 96880 71766 16029 27649 37640 47525 65884 29266 92410 99412 387 7918 90177 69729 55069 17262 49294 10385 55992 41845 6428 23736 3154 98200 644 11387 54173 51724 90349 4748 5160 7667 38015 90383 94041 35832 29958 82712 60415 13046 71589 22816 11896 83777 69403 86559 48897 69008 51254 72953 62956 35483 90783 34414 59039 93157 95977 18669 37884 82569 63088 3499 83469 46586 89792 34799 60510 67053 27192 32175 17263 83847 71448 86326 69852 68733 79262 25143 82761 74080 66733 68412 99654 89892 43522 63745 40633 27955 46271 30174 26237 54964 36941 19250 35410 47108 46415 32648 38148 65225 32886 11404 61749 41328 99537 36550 90986 89543 56744 94203 52727 49117 76152 44474 97537 17978 91164 95617 70148 80262 6255 73049 94458 63604 29862 51356 32140 91621 7038 73274 95555 39401 17822 71586 47616 70758 12419 71478 57003 15494 90482 75600 10514 16811 79609 76909 77434 33316 39134 78395 49079 65648 5304 22839 28775 70459 15527 14435 88006 41181 22073 6613 70395 40532 98466 19289 56068 2802 35298 79856 12874 25679 17299 35493 43404 91159 961 98266 74792 87314 5643 24641 43079 84809 47441 93815 35014 84746 43027 71913 41471 6373 79563 5932 82065 43143 76182 43596 97212 67055 75386 99116 15193 63702 77554 8708 16636 85946 59692 98778 19816 13391 97568 35080 15680 45483 94692 3596 4291 43766 81588 11547 608 57184 9766 62868 68095 50178 71121 52625 7708 20883 43656 15 44082 28414 20806 39607 10132 84936 32209 56100 87994 43846 89577 11987 68213 49534 9620 57508 43707 53926 70695 76079 53221 63676 55918 42557 40214 93882 66255 77386 69929 6374 76863 11216 77487 65424 34087 1951 17234 72597 20025 10767 78357 1800 85160 13383 65234 44136 13254 66184 41610 94679 20924 83271 37290 25969 63963 24048 58665 73227 15815 69523 92743 70633 43302 80475 32971 43475 84352 97791 66568 27438 30029 49528 17763 86531 97509 48590 76939 3727 2531 76654 94654 80805 18963 11553 23582 26712 32517 9963 11554 51904 83321 85538 16766 21240 52607 76162 55693 98554 98967 96091 95022 71980 40885 97789 96082 23386 93788 44928 1822 40109 88131 19326 40384 98104 35519 3567 31475 76956 73776 28166 29645 82522 48232 34003 56454 86476 21239 71419 98968 64976 52179 29082 61871 53761 93403 24203 44191 1642 99422 43422 98303 15917 61737 75288 11809 41533 68980 59605 55030 46600 45789 13233 55512 1478 54533 98725 62450 91345 480 71342 36175 10019 33434 58753 69179 37257 85039 62068 51670 64643 23178 6721 18731 26322 58475 64061 60387 81472 89809 110 89762 84328 31468 9849 14637 41897 4154 81717 96821 12637 90083 79312 49199 3897 11470 51903 25944 67230 63353 30665 85693 10660 13179 93905 31186 73948 44596 52559 22651 70844 73898 96262 50620 20773 8076 27768 52960 19435 92847 30501 64134 73492 30707 82199 49526 16069 56705 417 18012 37681 19333 2064 10684 87019 53664 89767 38569 47519 61875 65213 30719 92840 38494 32693 92115 5792 24695 11261 22278 47528 9126 21545 22396 83555 68183 33176 6955 95099 76114 85525 27237 17240 30393 53594 26015 34583 71492 94373 2009 90705 10448 53430 75162 13728 14341 90113 67001 14919 35254 29809 36199 4328 28410 43953 47588 79155 59107 52280 45978 45211 16887 91588 86124 82907 22782 99241 24176 16450 23393 97103 12586 30560 39421 61005 71022 10875 62520 66390 36259 17151 47331 46318 87178 85597 14146 29455 37824 23250 91430 17438 58614 49374 25855 5745 14506 63301 59748 24086 35653 17250 61652 132 10243 13511 45278 42496 40407 27288 50309 34567 9097 76519 40011 90466 69453 8757 88570 52974 83413 54708 26674 48824 42530 82101 40431 63675 21927 81404 60567 91891 10634 58956 76938 17844 98200 46075 61557 33615 68325 62560 59774 59883 45757 28436 3279 44425 76699 66717 41832 12189 21221 74960 8453 39938 90300 30708 74847 19835 26835 60311 81686 12746 48037 81329 44772 29666 46119 38106 96732 24755 29304 78244 32163 43140 71717 24201 92483 77572 73501 66643 64513 39521 89139 72345 51021 66932 27921 88565 40856 18431 44998 73483 99753 19000 94905 96773 42496 97575 18468 78451 23945 78466 88350 75870 53041 64648 95522 52039 85224 84769 52494 91487 3229 67278 90729 39607 15484 11193 72596 19518 94533 39464 50928 62874 31048 34531 2826 82871 25052 79892 9568 81004 58244 75737 88316 11745 80534 17435 46317 70790 38090 32755 70098 20584 49658 34297 70286 57746 88846 23146 206 9549 54948 87551 29515 51533 61125 55683 16555 10183 77713 19035 51003 62857 31057 52655 70155 25058 71268 34551 24625 24840 95634 29698 98661 66172 14886 13344 44344 1821 45337 551 2898 9124 96249 38398 36588 61235 23734 67499 49810 63608 6471 63838 68924 7869 87713 92116 66396 45912 92643 31739 25817 2456 46876 37597 12679 33855 90332 20049 13174 38687 18721 65146 10686 99839 89149 18455 27313 88617 32316 20299 68057 98070 39128 7933 42664 2024 15109 79010 45586 73899 82234 42087 91306 93727 15090 55617 34436 30330 7564 22956 26088 50849 53883 21666 49988 68709 16461 38287 8981 94582 56094 10213 93386 52297 98004 81244 37135 92827 64356 15875 51588 77069 40328 73170 81356 42740 70193 33987 3392 71435 10837 65158 91928 96594 91231 33404 70579 66886 16736 56792 50197 52186 89123 35627 43116 14927 71305 22196 95346 24998 29598 42947 32001 1205 54030 72983 89734 10193 52892 88408 21482 51447 47170 74932 4750 59578 74935 50937 27713 35721 75970 36293 45641 25136 48229 72402 71532 84026 27541 11629 19634 87362 22551 57583 30553 68268 62809 81394 59799 73058 88391 23191 11532 13269 24412 73160 34094 36420 52974 86307 33471 1263 68778 43737 22813 11765 90339 26863 77022 19929 97806 80200 23627 59727 21865 41087 33207 95579 49372 66232 57520 84424 53540 341 78950 61870 72329 770 90738 8480 21174 7789 80465 90973 35515 40204 78145 58132 17168 79383 29796 23080 42808 57963 34440 80489 83236 74640 27456 59392 67568 77644 17895 20077 14240 97179 32435 28594 12422 63442 56252 97305 59327 49504 59500 9333 39255 98793 37094 83145 70634 75088 22861 4904 8789 70207 83562 59323 45751 76983 53045 20324 41937 3804 4571 16764 76290 40389 21982 15939 72282 38338 28353 11295 71987 6770 9093 6983 4291 43826 63027 7312 5731 12578 50187 22584 68736 46910 32044 84220 82728 25545 89485 44568 92843 70847 4787 14103 45266 88105 53122 17011 11500 12513 4242 30769 41449 71304 62383 73028 76605 40708 72776 17001 59481 57752 11821 51063 56250 44451 10074 89007 16003 30999 47583 43708 87647 69743 75218 25382 64565 6349 27175 59279 37763 10387 9565 5733 6339 88119 52044 10305 95879 56808 15843 17416 96282 64099 56668 70571 85984 28267 55202 5937 8733 89687 16947 26981 14780 40019 95698 21392 42624 61168 3343 17538 35486 30040 2810 62827 28769 43644 48817 63794 26507 21759 3940 69484 31198 39963 16029 28157 28452 40142 18660 73598 59798 54918 16532 22857 27613 4122 95617 74715 7129 89936 26139 36970 96064 87105 14884 95784 79716 20635 25316 81290 27750 19307 43942 39430 32939 98313 90892 47895 11345 79424 23682 48746 80463 44663 31646 25514 95955 25988 69586 57423 91655 71396 85836 7011 99699 76394
74557 46345 69914 40692 16793 79402 5004 25291 356 3708 86760 63104 22821 97134 26296 25153 77475 81029 93395 55091 1537 14953 3866 69707 61210 86791 1308 99915 55607 78983 21042 87556 75254 70256 97630 5400 203 6187 98328 72244 807 65774 8212 19061 28514 553 56727 52943 64492 49479 10766 30287 37419 95616 86229 61762 57388 91478 78458 16485 3519 67718 71006 47596 39579 75339 8894 36388 17915 33889 30078 35732 95955 59099 6333 64691 46260 51608 7148 32003 37805 70249 49927 99952 75836 21155 63050 32096 35305 3898 20093 74383 26493 39918 34368 2062 57760 3690 20596 52927 16010 43815 88745 27096 37269 1109 3853 14849 24809 11483 47125 96273 29380 26165 48322 85932 84900 36697 85737 9177 56459 17853 43547 17904 25928 43950 11207 76924 26827 69082 25713 69187 93993 5510 11694 26557 92895 57955 79163 65699 79056 6788 95233 58567 44722 5732 89504 75470 49714 68486 6175 6017 65046 21738 69650 69000 87657 55117 48877 56697 64632 9818 55462 7238 75243 51034 23977 78078 86343 93462 42359 96619 84795 3223 89864 9355 79856 34556 28862 91603 51075 44008 94164 92134 27330
