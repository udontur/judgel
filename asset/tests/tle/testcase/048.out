7599594254366
